docs/extensions/guides/ipc.md

```diff
@@ -40,38 +40,46 @@ The general terminology is as follows:
 
 To register either a handler or a listener, you should do something like the following:
 
+File1:
```

Comment:
Can we call this `main.ts` instead of "File1" so it's clear this is code you could put on the `main` thread?